src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -186,10 +186,10 @@
     <value>If enabled, test that all color channels are within a tolerance distance from each other. Otherwise, check that the sum of all color channels differences is smaller than the tolerance.</value>
   </data>
   <data name="MeasureTool_DrawFeetOnCross.Header" xml:space="preserve">
-    <value>Draw feet on cross</value>
+    <value>Draw serif on cross</value>
```

Comment:
Isn't 'serif' related to typography specifically? In this context, I wouldn't know what it means.

Agree that 'feet' is also not great though haha.

Maybe something like "Add lines to cross"? (Also not great at all!)

Reply:
Hm, yeah, looks like you have a point about the typography. "Feet" is just too awkward (also get literal translated) and "lines on the cross" could also mean parallel lines like there already is. Difficult to find the right words...

Reply:
Maybe @crutkas has a good suggestion 😄

Reply:
Yeah, serif doesn't sound good here. I'd have no idea what that is about 😅 
I don't have better suggestions here, though.

Reply:
> a serif (/ˈsɛrɪf/) is a small line or stroke regularly attached to the end of a larger stroke

Reply:
I want to use Language that is quickly understandable. Serif is not one of those.  Do we have any data to confirm people have issue on that language?

Reply:
I don't have telemetry or anything, but I do remember [this comment](https://github.com/microsoft/PowerToys/pull/20214#issuecomment-1234114497). Seeing it literally translated is also awkward.